src/applications/vaos/routes.jsx

```diff
@@ -55,8 +57,12 @@ const ConfirmationPage = asyncLoader(() =>
 
 export default function createRoutesWithStore(store) {
   return (
-    <Route path="/" component={VAOSApp}>
-      <IndexRoute component={AppointmentsPage} />
+    <Route component={VAOSApp}>
+      <Route path="/" component={AppointmentsPage}>
+        <IndexRedirect to="upcoming" />
+        <Route component={FutureAppointmentsList} path="upcoming" />
```

Comment:
You could make this an `IndexRoute` and skip the redirect. The upcoming/past symmetry is nice, but the extra page could confuse GA and e2e tests and probably isn't worth it.